### PR TITLE
Use more helper methods on PageElement

### PR DESCRIPTION
**Pull request by camillobruni**
Passing in custom event classes to PageElement might become problematic for custom runners.
Let's use custom helper-methods for now to stay runner independent.

- Add mouseUp, mouseDown, mouseMove and scroll helpers
- Use deltaX instead of just delta for wheel events

### CODE REVIEW

File: resources/benchmark-runner.mjs [hunk 1]
```diff
@@ -76,20 +76,59 @@ class PageElement {
...
+        this.#node.focus();
+    }
+
     click() {
         this.#node.click();
     }
 
-    focus() {
-        this.#node.focus();
+    mouseMove(clientX = 0, clientY = 0) {
```
Comment (squarewave) [inline resources/benchmark-runner.mjs:87]: Nit: why the default values?

**Review by squarewave — APPROVED** [review]
This looks good to me. Thanks!

File: resources/tests.mjs [hunk 2]
```diff
@@ -395,51 +370,26 @@ Suites.push({
...
-            event = {
-                clientX: 200,
-                clientY: 200,
-                deltaMode: 0,
-                delta: -10,
-                deltaY: -10,
-                bubbles: true,
-                cancelable: true,
-            };
+                cursor.wheel(10, 10, 650, 200);
```
Comment (rniwa) [inline resources/tests.mjs:390]: Did you mean to call scroll? I don't think there is wheel method defined anywhere.

**Comment by rniwa** [comment]
I'm adding similar helper methods in https://github.com/WebKit/Speedometer/pull/156.

**Pull request closed without merging**